crates/precompile/Cargo.toml

```diff
@@ -118,6 +126,9 @@ libsecp256k1 = ["dep:libsecp256k1"]
 # Enables the BLS12-381 precompiles.
 blst = ["dep:blst"]
 
+# Use matter-labs implementation for BN128 operations (EIP1962)
+matter-labs-eip1962 = ["dep:eth_pairings"]
```

Comment:
matter-labs-eip1962 is a pretty long name, perhaps we can make it shorter

Reply:
It is fine, it is better to be descriptive